Adding `meta_desc` to blogs (first batch) (#1499)
pulumi/docs: content/blog/announcing-support-for-email-based-identities/index.md

title: "Announcing Support for Email-based Identities"
authors: ["praneet-loke"]
tags: ["Pulumi-News","Features"]

> meta_desc: "Pulumi now supports for email-based identities, on top of support for Atlassian identity and GitLab."

Reviewer: ```suggestion
meta_desc: "Pulumi now supports email-based identities, in addition to GitHub, Atlassian, and GitLab."
```
